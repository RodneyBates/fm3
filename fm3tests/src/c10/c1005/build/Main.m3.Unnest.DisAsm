   414 16_00000000000000a2                   162 ItkEOF()
   412 16_00000000000000a4                   164 ItkRightEnd()
   410 16_00000000000000ec                   236 ItkBlockBEGINPatch(C388,L1(16_1),[13,2])
   403 16_00000000000000e9                   233 ItkBlockRt(L1(16_1),[3,2])
   398 16_00000000000001a9                   425 ItkStmtListLt(L0(16_0),[14,2])
   393 16_00000000000001ac                   428 ItkStmtListRt(L0(16_0),[14,2])
   388 16_00000000000001a3                   419 ItkBlockDeclListLtPatch(C24,L1(16_1),[3,2])
   382 16_00000000000001a4                   420 ItkBlockDeclListRt(L1(16_1),[3,2])
   377 16_00000000000000dc                   220 ItkProcWBodyLtPatch(C24,L2(16_2))
   373 16_00000000000000dd                   221 ItkProcWBodyRt(L2(16_2))
   370 16_00000000000000e0                   224 ItkProcBodyLtPatch(C323,[10,1])
   364 16_00000000000000e1                   225 ItkProcBodyRt([10,1])
   360 16_00000000000000ec                   236 ItkBlockBEGINPatch(C338,L4(16_4),[10,3])
   353 16_00000000000000e9                   233 ItkBlockRt(L4(16_4),[10,3])
   348 16_00000000000001a9                   425 ItkStmtListLt(L0(16_0),[11,3])
   343 16_00000000000001ac                   428 ItkStmtListRt(L0(16_0),[11,3])
   338 16_00000000000001a1                   417 ItkBlockDeclListLt(L0(16_0),[10,3])
   333 16_00000000000001a4                   420 ItkBlockDeclListRt(L0(16_0),[10,3])
   328 16_00000000000000e6                   230 ItkBlockLt(L4(16_4),[10,3])
   323 16_00000000000000c4                   196 ItkProcSignatureLtPatch(C24,[4,5])
   318 16_00000000000000c5                   197 ItkProcSignatureRt([4,5])
   314 16_00000000000000ae                   174 ItkRaisesSetAbsent([10,1])
   310 16_00000000000000b0                   176 ItkResultTypeAbsent([10,1])
   306 16_0000000000000183                   387 ItkFormalsListLtPatch(C24,L6(16_6),[4,5])
   300 16_0000000000000184                   388 ItkFormalsListRt(L6(16_6),[4,5])
   295 16_0000000000000187                   391 ItkFormalsListSepPatch(C241,L5(16_5),[8,3])
   288 16_0000000000000135                   309 ItkVALUEFormalLtPatch(C241,[8,5])
   282 16_0000000000000139                   313 ItkVALUEFormalTypePatch(C258,[9,3])
   276 16_000000000000013c                   316 ItkVALUEFormalValPatch(C262,[9,3])
   270 16_0000000000000136                   310 ItkVALUEFormalRt([8,5])
   266 16_00000000000000ad                   173 ItkFormalExprAbsent([9,3])
   262 16_00000000000000ac                   172 ItkFormalTypeAbsent([9,3])
   258 16_000000000000018b                   395 ItkVALUEFormalIdListLtPatch(C241,L1(16_1),[8,5])
   251 16_000000000000018c                   396 ItkVALUEFormalIdListRt(L1(16_1),[8,5])
   246 16_0000000000000190                   400 ItkVALUEFormalIdListElem(Id9"F6",[8,5])
   241 16_0000000000000187                   391 ItkFormalsListSepPatch(C186,L4(16_4),[7,3])
   234 16_000000000000013f                   319 ItkVARFormalLtPatch(C186,[7,9])
   228 16_0000000000000143                   323 ItkVARFormalTypePatch(C203,[7,12])
   222 16_0000000000000146                   326 ItkVARFormalValPatch(C208,[8,3])
   216 16_0000000000000140                   320 ItkVARFormalRt([7,5])
   212 16_00000000000000ad                   173 ItkFormalExprAbsent([8,3])
   208 16_00000000000000b5                   181 ItkIdRefAtom(Id8"IDECHAR",[7,14])
   203 16_0000000000000193                   403 ItkVARFormalIdListLtPatch(C186,L1(16_1),[7,9])
   196 16_0000000000000194                   404 ItkVARFormalIdListRt(L1(16_1),[7,9])
   191 16_0000000000000198                   408 ItkVARFormalIdListElem(Id7"F5",[7,9])
   186 16_0000000000000187                   391 ItkFormalsListSepPatch(C130,L3(16_3),[6,3])
   179 16_0000000000000149                   329 ItkROFormalLtPatch(C130,[6,14])
   173 16_000000000000014d                   333 ItkROFormalTypePatch(C147,[6,17])
   167 16_0000000000000150                   336 ItkROFormalValPatch(C153,[7,3])
   161 16_000000000000014a                   330 ItkROFormalRt([6,5])
   157 16_00000000000000ad                   173 ItkFormalExprAbsent([7,3])
   153 16_00000000000000b4                   180 ItkIdReserved(Rid"BOOLEAN",[6,19])
   147 16_000000000000019b                   411 ItkROFormalIdListLtPatch(C130,L1(16_1),[6,14])
   140 16_000000000000019c                   412 ItkROFormalIdListRt(L1(16_1),[6,14])
   135 16_00000000000001a0                   416 ItkROFormalIdListElem(Id6"F4",[6,14])
   130 16_0000000000000187                   391 ItkFormalsListSepPatch(C74,L2(16_2),[5,3])
   123 16_0000000000000135                   309 ItkVALUEFormalLtPatch(C74,[5,11])
   117 16_0000000000000139                   313 ItkVALUEFormalTypePatch(C91,[5,14])
   111 16_000000000000013c                   316 ItkVALUEFormalValPatch(C97,[6,3])
   105 16_0000000000000136                   310 ItkVALUEFormalRt([5,5])
   101 16_00000000000000ad                   173 ItkFormalExprAbsent([6,3])
    97 16_00000000000000b4                   180 ItkIdReserved(Rid"CHAR",[5,16])
    91 16_000000000000018b                   395 ItkVALUEFormalIdListLtPatch(C74,L1(16_1),[5,11])
    84 16_000000000000018c                   396 ItkVALUEFormalIdListRt(L1(16_1),[5,11])
    79 16_0000000000000190                   400 ItkVALUEFormalIdListElem(Id5"F3",[5,11])
    74 16_0000000000000135                   309 ItkVALUEFormalLtPatch(C24,[4,7])
    69 16_0000000000000139                   313 ItkVALUEFormalTypePatch(C45,[4,15])
    64 16_000000000000013c                   316 ItkVALUEFormalValPatch(C51,[5,3])
    59 16_0000000000000136                   310 ItkVALUEFormalRt([4,7])
    55 16_00000000000000ad                   173 ItkFormalExprAbsent([5,3])
    51 16_00000000000000b4                   180 ItkIdReserved(Rid"INTEGER",[4,17])
    45 16_000000000000018b                   395 ItkVALUEFormalIdListLtPatch(C24,L2(16_2),[4,7])
    39 16_000000000000018c                   396 ItkVALUEFormalIdListRt(L2(16_2),[4,7])
    34 16_0000000000000190                   400 ItkVALUEFormalIdListElem(Id4"F2",[4,12])
    29 16_0000000000000190                   400 ItkVALUEFormalIdListElem(Id3"F1",[4,7])
    24 16_00000000000000e6                   230 ItkBlockLt(L1(16_1),[3,2])
    19 16_0000000000000154                   340 ItkExportIdListRt(L1(16_1),[1,7])
    14 16_0000000000000158                   344 ItkExportIdListElem(Id1"Main",[1,7])
     9 16_0000000000000151                   337 ItkExportIdListLt(L1(16_1),[1,7])
     4 16_00000000000000a3                   163 ItkLeftEnd()
     2 16_00000000000000a1                   161 ItkBOF()
     0
