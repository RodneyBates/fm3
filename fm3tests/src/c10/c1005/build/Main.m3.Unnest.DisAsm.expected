   393 16_00000000000000a2                   162 ItkEOF()
   391 16_00000000000000a4                   164 ItkRightEnd()
   389 16_00000000000000ec                   236 ItkBlockBEGINPatch(C367,L1(16_1),[13,2])
   382 16_00000000000000e9                   233 ItkBlockRt(L1(16_1),[3,2])
   377 16_00000000000001c7                   455 ItkStmtListLt(L0(16_0),[14,2])
   372 16_00000000000001ca                   458 ItkStmtListRt(L0(16_0),[14,2])
   367 16_00000000000001c1                   449 ItkBlockDeclListLtPatch(C24,L1(16_1),[3,2])
   361 16_00000000000001c2                   450 ItkBlockDeclListRt(L1(16_1),[3,2])
   356 16_00000000000000dc                   220 ItkProcWBodyLtPatch(C24,L2(16_2))
   352 16_00000000000000dd                   221 ItkProcWBodyRt(L2(16_2))
   349 16_00000000000000e0                   224 ItkProcBodyLtPatch(C302,[10,1])
   343 16_00000000000000e1                   225 ItkProcBodyRt([10,1])
   339 16_00000000000000ec                   236 ItkBlockBEGINPatch(C317,L4(16_4),[10,3])
   332 16_00000000000000e9                   233 ItkBlockRt(L4(16_4),[10,3])
   327 16_00000000000001c7                   455 ItkStmtListLt(L0(16_0),[11,3])
   322 16_00000000000001ca                   458 ItkStmtListRt(L0(16_0),[11,3])
   317 16_00000000000001bf                   447 ItkBlockDeclListLt(L0(16_0),[10,3])
   312 16_00000000000001c2                   450 ItkBlockDeclListRt(L0(16_0),[10,3])
   307 16_00000000000000e6                   230 ItkBlockLt(L4(16_4),[10,3])
   302 16_00000000000000c4                   196 ItkProcSignatureLtPatch(C24,[4,4])
   297 16_00000000000000c5                   197 ItkProcSignatureRt([4,4])
   293 16_00000000000000ae                   174 ItkRaisesSetAbsent([10,1])
   289 16_00000000000000b0                   176 ItkResultTypeAbsent([10,1])
   285 16_00000000000001a1                   417 ItkFormalsListLtPatch(C24,L5(16_5),[4,4])
   279 16_00000000000001a2                   418 ItkFormalsListRt(L5(16_5),[4,4])
   274 16_00000000000001a5                   421 ItkFormalsListSepPatch(C242,L5(16_5),[8,4])
   267 16_00000000000000ad                   173 ItkFormalExprAbsent([9,4])
   263 16_00000000000000ac                   172 ItkFormalTypeAbsent([9,4])
   259 16_00000000000001a9                   425 ItkVALUEFormalIdListLtPatch(C242,L1(16_1),[8,6])
   252 16_00000000000001aa                   426 ItkVALUEFormalIdListRt(L1(16_1),[8,6])
   247 16_00000000000001ae                   430 ItkVALUEFormalIdListElem(Id8"F6",[8,6])
   242 16_00000000000001a5                   421 ItkFormalsListSepPatch(C186,L4(16_4),[7,4])
   235 16_0000000000000155                   341 ItkVARFormalLtPatch(C186,[7,10])
   229 16_0000000000000159                   345 ItkVARFormalTypePatch(C203,[7,13])
   223 16_000000000000015c                   348 ItkVARFormalValPatch(C209,[8,4])
   217 16_0000000000000156                   342 ItkVARFormalRt([7,6])
   213 16_00000000000000ad                   173 ItkFormalExprAbsent([8,4])
   209 16_00000000000000b4                   180 ItkReservedId(Rid"WIDECHAR",[7,15])
   203 16_00000000000001b1                   433 ItkVARFormalIdListLtPatch(C186,L1(16_1),[7,10])
   196 16_00000000000001b2                   434 ItkVARFormalIdListRt(L1(16_1),[7,10])
   191 16_00000000000001b6                   438 ItkVARFormalIdListElem(Id7"F5",[7,10])
   186 16_00000000000001a5                   421 ItkFormalsListSepPatch(C130,L3(16_3),[6,4])
   179 16_000000000000015f                   351 ItkROFormalLtPatch(C130,[6,15])
   173 16_0000000000000163                   355 ItkROFormalTypePatch(C147,[6,18])
   167 16_0000000000000166                   358 ItkROFormalValPatch(C153,[7,4])
   161 16_0000000000000160                   352 ItkROFormalRt([6,6])
   157 16_00000000000000ad                   173 ItkFormalExprAbsent([7,4])
   153 16_00000000000000b4                   180 ItkReservedId(Rid"BOOLEAN",[6,20])
   147 16_00000000000001b9                   441 ItkROFormalIdListLtPatch(C130,L1(16_1),[6,15])
   140 16_00000000000001ba                   442 ItkROFormalIdListRt(L1(16_1),[6,15])
   135 16_00000000000001be                   446 ItkROFormalIdListElem(Id6"F4",[6,15])
   130 16_00000000000001a5                   421 ItkFormalsListSepPatch(C74,L2(16_2),[5,4])
   123 16_000000000000014b                   331 ItkVALUEFormalLtPatch(C74,[5,12])
   117 16_000000000000014f                   335 ItkVALUEFormalTypePatch(C91,[5,15])
   111 16_0000000000000152                   338 ItkVALUEFormalValPatch(C97,[6,4])
   105 16_000000000000014c                   332 ItkVALUEFormalRt([5,6])
   101 16_00000000000000ad                   173 ItkFormalExprAbsent([6,4])
    97 16_00000000000000b4                   180 ItkReservedId(Rid"CHAR",[5,17])
    91 16_00000000000001a9                   425 ItkVALUEFormalIdListLtPatch(C74,L1(16_1),[5,12])
    84 16_00000000000001aa                   426 ItkVALUEFormalIdListRt(L1(16_1),[5,12])
    79 16_00000000000001ae                   430 ItkVALUEFormalIdListElem(Id5"F3",[5,12])
    74 16_000000000000014b                   331 ItkVALUEFormalLtPatch(C24,[4,6])
    69 16_000000000000014f                   335 ItkVALUEFormalTypePatch(C45,[4,14])
    64 16_0000000000000152                   338 ItkVALUEFormalValPatch(C51,[5,4])
    59 16_000000000000014c                   332 ItkVALUEFormalRt([4,6])
    55 16_00000000000000ad                   173 ItkFormalExprAbsent([5,4])
    51 16_00000000000000b4                   180 ItkReservedId(Rid"INTEGER",[4,16])
    45 16_00000000000001a9                   425 ItkVALUEFormalIdListLtPatch(C24,L2(16_2),[4,6])
    39 16_00000000000001aa                   426 ItkVALUEFormalIdListRt(L2(16_2),[4,6])
    34 16_00000000000001ae                   430 ItkVALUEFormalIdListElem(Id4"F2",[4,11])
    29 16_00000000000001ae                   430 ItkVALUEFormalIdListElem(Id3"F1",[4,6])
    24 16_00000000000000e6                   230 ItkBlockLt(L1(16_1),[3,2])
    19 16_000000000000016a                   362 ItkExportIdListRt(L1(16_1),[1,7])
    14 16_000000000000016e                   366 ItkExportIdListElem(Id1"Main",[1,7])
     9 16_0000000000000167                   359 ItkExportIdListLt(L1(16_1),[1,7])
     4 16_00000000000000a3                   163 ItkLeftEnd()
     2 16_00000000000000a1                   161 ItkBOF()
     0
